packages/calling-stateful-client/src/Converter.ts

```diff
@@ -155,7 +167,10 @@ export function convertSdkCallToDeclarativeCall(call: CallCommon): CallState {
     /* @conditional-compile-remove(optimal-video-count) */
     optimalVideoCount: {
       maxRemoteVideoStreams: call.feature(Features.OptimalVideoCount).optimalVideoCount
-    }
+    },
+    /* @conditional-compile-remove(hide-attendee-name) */
+    // TODO: Replace this once the SDK supports hide attendee name
+    hideAttendeeNames: false
```

Comment:
how are we going to set true/false at the API layer? it looks like its only false for now? Do we have a plan?

Reply:
Yeas, we are going to add the flag for `hideAttendeeName` in the call object from SDK side and we can use that flag here for the call data. Right now setting it to false and using this hardcoded prop to test the changes